src/theme/foundations/textStyles.ts

```diff
@@ -1,51 +1,95 @@
 export const textStyles = {
   h1: {
-    fontFamily: '"JetBrains Mono", monospace',
+    fontFamily: 'heading',
     fontWeight: 700,
     fontSize: '2.75rem',
     lineHeight: '3.375rem',
     letterSpacing: '5%',
     color: 'body'
   },
   h2: {
-    fontFamily: '"JetBrains Mono", monospace',
+    fontFamily: 'heading',
     fontWeight: 400,
     fontSize: '1.5rem',
     lineHeight: 'auto',
     letterSpacing: '4%',
     color: 'body'
   },
+  homepageH1: {
+    fontFamily: 'heading',
+    fontWeight: { base: 700, md: 500 },
+    fontSize: { base: '2.75rem', md: '6rem' },
+    outline: '1px solid red',
+    lineHeight: { base: '3.375rem', md: '6rem' }
+  },
+  header1: {
+    fontFamily: 'heading',
+    fontWeight: 700,
+    fontSize: '2.125rem',
+    letterSpacing: '3%',
```

Comment:
```suggestion
    letterSpacing: '4%',
    lineHeight: '44.88px'
```

Just updating with figma values
<img width="933" alt="Screen Shot 2022-11-28 at 12 21 10 PM" src="https://user-images.githubusercontent.com/15589226/204362846-e2c27869-24b1-42a0-83bb-4686d25e55b0.png">

Reply:
I'm not sure about this
<img width="949" alt="image" src="https://user-images.githubusercontent.com/54227730/204381906-4c3fa762-107b-457f-acd0-26f16ed4e4d6.png">

The `lineHeight` in the CSS is suggesting `45px`.

<img width="338" alt="image" src="https://user-images.githubusercontent.com/54227730/204382470-c6ac62c5-351c-4341-849b-4ff76beb5cf5.png">

Inspecting this, `auto` and `3%` appear to be invalid values for those css properties.

When I check `lineHeight: 45px` it's the same as using `lineHeight: normal`, which I feel like this might be the more appropriate way to declare it, which should match the box height.

The letter spacing as well, realizing `%` doesn't work. So instead of `4%` here we should use `0.04em`

Reply:
Just pushed a commit that updates these values to use "normal" instead of "auto" for `lineHeight`, and to use `em` units instead of `%` for `letterSpacing`. Updated for `header1` to use `0.04em` instead of 3%